core/lib/spree/testing_support/dummy_app.rb

```diff
@@ -134,6 +134,7 @@ class Application < ::Rails::Application
 Spree.load_defaults(Spree.solidus_version)
 Spree.config do |config|
   config.mails_from = "store@example.com"
+  config.associate_user_in_authentication_extension = false
```

Comment:
I guess we can remove this line. If we use the defaults by version, it'll use `true` on the current version so no warning will be emitted.